ariadne/cardano/src/Ariadne/Wallet/Cardano/Kernel/Keystore.hs

```diff
@@ -263,3 +193,7 @@ toList (Keystore ks) =
 
 walletIdToKey :: WalletId -> AddressHash PublicKey
```

Comment:
Btw, I have a similar question to the one I had about `eskToKey`. It returns a hash, not a key.

Reply:
I changed the name accordingly.